src/core/Akka/Actor/Internal/ActorSystemImpl.cs

```diff
@@ -302,13 +337,13 @@ private void StopScheduler()
             sched?.Dispose();
         }
 
-        private void LoadExtensions()
+        private List<object> LoadExtensions()
```

Comment:
Can we type this usefully in order to avoid boxing?

Reply:
This are only the extensions loaded from config, excluding lazy loaded extensions